themes.json

```diff
@@ -307,6 +307,18 @@
       "display": "psyneulink-sphinx-theme",
       "pypi": "psyneulink-sphinx-theme"
     },
+    {
+      "config": {
+        "_imports": [
+          "pylons_sphinx_themes"
+        ],
+        "html_sidebars": "{ '**': [\n    'localtoc.html',\n    'ethicalads.html',\n    'relations.html',\n    'sourcelink.html',\n    'searchbox.html',\n] }",
```

Comment:
Why not let the theme use it's default `html_sidebars`?

Reply:
because it is [recommended](https://github.com/Pylons/pylons-sphinx-themes/blob/master/README.rst#edit-your-sphinxs-confpy). but, hmm, no need to enable "Ethical Ads" in this site. I will delete, thanks.

and I found this theme has another themes. I will add those in this PR.

Reply:
Yea, we don't embed ethicalads scripts, so it's a no-op for us. Since dropping this won't change anything, it's fine to drop it.

Reply:
> and I found this theme has another themes. I will add those in this PR.

Please don't? Those are specific to the projects named there, and not meant to be reusable.

Reply:
The one-theme-per-package is also a specific design choice (there's a bunch of hoops to jump to allow for that which feels unnecessary) and I'd rather we nudge downstream authors to, well, have one theme per package.

From https://github.com/sphinx-themes/sphinx-themes.org/blob/master/CONTRIBUTING.md#adding-a-new-theme:

> Every theme must be available as a PyPI package, and only 1 entry is permitted for a package.

Reply:
Oh, sorry, I didn't noticed that. but why not? multiple themes in one package is a Sphinx's design choice.

Reply:
> but why not?

Sure, but no good-looking reusable theme does this. We can revisit and explore adding this complexity if such a theme is ever created.

> multiple themes in one package is a Sphinx's design choice.

Well, yes. But until some "relevant" theme uses it, this is unnecessary complexity.

Reply:
I have reverted the this multiple themes code, because it is not a target of this PR. Instead, I opened #63. so let's track on the issue.